### PR TITLE
test: add E2E tests for ManagerDialog

### PR DESCRIPTION
**Pull request by dante01yoon**
## Summary
- Add Playwright E2E tests for the ManagerDialog component which had zero test coverage
- Covers dialog opening, pack browsing, search filtering, tab navigation, sort controls, search mode switching, error states, install action, info panel, and close behavior
- All API calls (Algolia search, Manager installed packs, queue) are mocked with typed responses

Part of the FixIt Burndown test coverage initiative.

## Test plan
- [ ] CI browser tests pass
- [ ] Tests validate core ManagerDialog user flows with mocked APIs
- [ ] No regressions in existing tests

┆Issue is synchronized with this [Notion page](https://www.notion.so/PR-10970-test-add-E2E-tests-for-ManagerDialog-33c6d73d365081468a4ad8fc1894f05b) by [Unito](https://www.unito.io)

### CODE REVIEW

File: browser_tests/tests/dialogs/managerDialog.spec.ts [hunk 1]
```diff
@@ -0,0 +1,417 @@
...
+  ]
+}
+
+test.describe('ManagerDialog', { tag: '@ui' }, () => {
+  test.beforeEach(async ({ comfyPage }) => {
+    const statsWithManager = {
+      ...mockSystemStats,
+      system: {
+        ...mockSystemStats.system,
+        argv: ['main.py', '--listen', '0.0.0.0', '--enable-manager']
```
Comment (AustinMroz) [inline browser_tests/tests/dialogs/managerDialog.spec.ts:170]:
```suggestion
        argv: ['main.py', '--enable-manager']
```
Very nit: `--listen` isn't required for the functionality of the test. Given its usage comes with some heavy security concerns, I'd prefer we avoid having tests that present it as being a reasonable default.

**Review by AustinMroz — APPROVED** [review]
Mocking is a little heavier than I'd like, but I'm not seeing an obviously nicer way to go about things.

File: browser_tests/tests/dialogs/managerDialog.spec.ts (hunk 1 above)
Comment (dante01yoon) [reply browser_tests/tests/dialogs/managerDialog.spec.ts:170]: @coderabbitai create issue for this and assign me

**Comment by dante01yoon** [comment]
@coderabbitai approve
